docs/spec/src/integration/spec/5-lifecycle-phases.md

```diff
@@ -0,0 +1,169 @@
+
+**QuorumNumbers**
+
+QuorumNumbers represents a list a quorums that are required to sign over and make the blob available. The quorums 0 represents the ETH quorum, and quorum 1 represents the EIGEN quorum, and both of these are required. Custom quorums can also be added to this list.
+
+**BlobCommitment**
+
+The BlobCommitment is the unique identifier for an EigenDA Blob. It can either be computed locally from the blob, or one can ask the disperser to generate it via the `GetBlobCommitment` endpoint.
+
+```protobuf
+message BlobCommitment {
+  // A G1 commitment to the blob data.
+  bytes commitment = 1;
+  // A G2 commitment to the blob data.
+  bytes length_commitment = 2;
+    // Used with length_commitment to assert the correctness of the `length` field below.
+  bytes length_proof = 3;
+  // Length in bn254 field elements (32 bytes) of the blob. Must be a power of 2.
+  uint32 length = 4;
+}
+```
+
+Unlike Ethereum blobs which are all 128KiB, EigenDA blobs can be any power of 2 length between 32KiB and 16MiB (currently), and so the `commitment` alone is not sufficient to prevent certain attacks:
+
+- Why is a commitment to the length of the blob necessary?
+    
+    There are different variants of the attack. The basic invariant the system needs to satisfy is that with the chunks from sufficient set of validators, you can get back the full blob. So the total size of the chunks held by these validators needs to exceed the blob size. If I don't know the blob size (or at least an upper bound), there's no way for the system to validate this invariant.
+    Here’s a simple example. Assume a network of 8 DA nodes, and coding ratio 1/2. For a `blob` containing 128 field elements (FEs), each node gets 128*2/8=32 FEs, meaning that any 4 nodes can join forces and reconstruct the data. Now assume a world without length proof; a malicious disperser receives the same blob, uses the same commitment, but claims that the blob only had length 4 FEs. He sends each node 4*2/8=1 FE. The chunks submitted to the nodes match the commitment, so the nodes accept and sign over the blob’s batch. But now there are only 8 FEs in the system, which is not enough to reconstruct the original blob (need at least 128 for that).
+    
+
+> Note that the length here is the length of the blob (power of 2), which is different from the payload_length encoded as part of the `PayloadHeader` in the `blob` itself (see the [encoding section](#encoding)).
+> 
+
+**PaymentHeader**
+
+The paymentHeader specifies how the blob dispersal to the network will be paid for. There are 2 main modes of payment, the permissionless pay-per-blob model and the permissioned reserved-bandwidth approach. See the [Payments](https://docs.eigenda.xyz/releases/payments) release doc for full details; we will only describe how to set these 4 fields here.
```

Comment:
There are *only* these modes of payment, right?

There are 2 ~~main~~ modes of payment

Reply:
[176dd65](https://github.com/Layr-Labs/eigenda/pull/1439/commits/176dd652e57e14075b2e32d93af34488a4fff2d9)